backend/src/Civix/ApiBundle/Controller/GroupController.php

```
@@ -69,9 +70,27 @@ public function createGroupAction(Request $request)
         $encodedPassword = $encoder->encodePassword($password, $group->getSalt());
         $group->setPassword($encodedPassword)->setOwner($this->getUser());
 
-        $em->persist($group);
-        $em->flush($group);
+        $slugify = new Slugify();
 
+        $groupName = $slugify->slugify($group->getOfficialName());
+
+        $mailgun = $this->get('civix_core.mailgun')->listcreateAction($groupName,$group->getOfficialDescription());
+
+        if($mailgun['http_response_code'] != 200){
```

when mailgun has an issue it should return an error to the end user and that's why i'm testing if it's not a 200 OK i'll return an error